### DISCUSSION TITLE
DB2 Database instrumentation support

### DISCUSSION THREAD

**Discussion by venkatramanchandirasekaran** [open-telemetry/opentelemetry-dotnet#3165] [Q&A]
Does the Open Telemetry Dotnet SDK support any other databases other than SQL server ? .

Looking at the SDK code there is only an instrumentation support for SQL Server via [OpenTelemetry.Instrumentation.SqlClient](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.SqlClient).

If there is no support for DB2 at this time, are there plans to add support for DB2 and if so what are the timelines ?

**Comment by alanwest** [comment]
You can find additional instrumentation libraries in the contrib repo. These are instrumentations contributed by the community at large: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src.

I am not aware of anyone working on instrumentation for DB2, but we always welcome new contributors!

**Comment by venkatramanchandirasekaran** [comment]
Thanks a lot.
